osd_planning/gcp-ccs.adoc

```diff
@@ -15,8 +15,11 @@ include::modules/ccs-gcp-customer-procedure.adoc[leveloffset=+1]
 include::modules/ccs-gcp-iam.adoc[leveloffset=+1]
 include::modules/ccs-gcp-provisioned.adoc[leveloffset=+1]
 include::modules/gcp-limits.adoc[leveloffset=+1]
+include::modules/osd-gcp-psc-firewall-prerequisites.adoc[leveloffset=+1]
 
 [id="additional-resources_{context}"]
 == Additional resources
 
-* xref:../security/rh-required-whitelisted-IP-addresses-for-sre-access.adoc#rh-required-whitelisted-IP-addresses-for-sre-access[Required allowlist IP addresses for SRE access]
\ No newline at end of file
+* xref:../security/rh-required-whitelisted-IP-addresses-for-sre-access.adoc#rh-required-whitelisted-IP-addresses-for-sre-access[Required allowlist IP addresses for SRE access]
```

Comment:
This should be removed from this page as it is not applicable. I'm not actually sure the purpose of that page at all, as it also doesn't apply to OSD on AWS...

Reply:
Hi @michaelryanmcneill, Did some research and came up with this doc sticket: https://issues.redhat.com/browse/OSDOCS-11353
I can/will remove link from page but wait until you get a chance to look at the docs ticket to verify if you believe this was added erroneously before making additional changes. If we do decide to remove topic entirely, will need to open an additional PR for that. thx. Thanks.

Reply:
Opened a thread in Slack to discuss this further, will circle back shortly!

Reply:
Created a docs ticket to address this issue once we determine best path forward. Thanks! https://issues.redhat.com/browse/OSDOCS-12438